Mod2/Lessons/Week4/CRUDwithORM.md

```diff
@@ -0,0 +1,202 @@
+# CRUD with ORM
+
+## Intro
+
+Up to this point, you've created a database using PostGreSQL and pgAdmin. You've used CRUD operations to manipulate your database:
```

Comment:
This is a super small thing, I've only seen it as PostgreSQL without the capital G. But this also totally might be another way I just haven't seen yet.